sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/documentanalysis/DocumentAnalysisAsyncClientTest.java

```diff
@@ -1205,7 +1195,7 @@ public void testGermanDocumentLanguagePrebuiltRead(HttpClient httpClient,
                 .setPollInterval(durationTestMode).getSyncPoller();
             AnalyzeResult analyzeResult = syncPoller.getFinalResult();
             Assertions.assertNotNull(analyzeResult);
-            Assertions.assertNotNull("de", analyzeResult.getLanguages().get(0).getLocale());
+            Assertions.assertEquals("de", analyzeResult.getLanguages().get(0).getLocale());
```

Comment:
Not sure if this will need to be re-recorded as this is now checking the locale properly